radixdlt/src/main/java/com/radixdlt/consensus/liveness/ProposerElection.java

```diff
@@ -31,4 +31,11 @@
 	 * @return Whether the node is a valid proposer
 	 */
 	boolean isValidProposer(EUID nid, View view);
+
+	/**

```

Comment:
Do we want to think about primary/secondary proposers? 
If we only want to deal with primary proposers, having both isValidProposer and getProposer is redundant.